### ISSUE TITLE
trying to run plot example but problems

### ISSUE THREAD

**Issue by sbillinge** [xpdAcq/streamz_ext#22]
Hi @CJ-Wright 

We are trying to build a pipeline from the ``live_plot.py`` example in streams_ext but running into tricky issues..... it won't import ``node_style`` and ``run_viz`` even though it can find and import the module in an ipython session in the conda env.  Any help appreciate.....

S

**Comment by sbillinge** [comment]
here is the stack dump btw:
```
$ python live_plot.py
Traceback (most recent call last):
  File "live_plot.py", line 2, in <module>
    from streamz_ext.graph import node_style, run_vis
ImportError: cannot import name 'node_style'
```

**Comment by CJ-Wright** [comment]
Have you installed from source? I haven't released a new version yet.

**Comment by sbillinge** [comment]
ah, ok. will do.

**Comment by sbillinge** [comment]
quick question...is the flow:
1. conda uninstall streamz_ext
1. python setupy.py develop

all in the conda env?

**Comment by CJ-Wright** [comment]
Yep that should work. You will also need to install grave. The score.yaml has the command under run, grave

**Comment by sbillinge** [comment]
having trouble finding that but I think it was a conda install

**Comment by CJ-Wright** [comment]
I don't think grave is on conda, you will need to install it from source.
